mix.exs

```diff
@@ -41,8 +41,10 @@ defmodule SpandexTesla.MixProject do
   defp deps do
     [
       {:spandex, "~> 3.0", optional: true},
+      {:tesla, ">= 1.3.3", optional: true},
       {:mox, "~> 0.5", only: :test},
-      {:ex_doc, "~> 0.21.3", only: :dev, runtime: false}
+      {:ex_doc, "~> 0.21.3", only: :dev, runtime: false},
+      {:credo, "~> 1.5", only: [:dev, :test], runtime: false}
```

Comment:
I would refrain form adding `credo` as a dependency. Maybe in a different PR so we can propertly set it up the CI for it

Reply:
Makes sense! I'll remove it.